api/src/migrations/api/1621963684141-ProjectSpecificWdpaEvents.ts

```diff
@@ -0,0 +1,18 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class CostSurfaceEvents1621963684141 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+      INSERT INTO api_event_kinds (id) values
+      ('project.wdpa.submitted/v1/alpha'),
+      ('project.wdpa.finished/v1/alpha'),
+      ('project.wdpa.failed/v1/alpha');
```

Comment:
I suggest to use *protected areas* rather than (the more specific) *wdpa* term where relevant - I know our core db table for protected areas is called `wdpa` but in practice it's a table for protected area info, some (most in some cases?) of which happen to be from the WDPA source. We've discussed this with Alicia at some point in the past, though we have kept the `wdpa` table name and some other such references around.

```suggestion
      ('project.protectedAreas.submitted/v1/alpha'),
      ('project.protectedAreas.finished/v1/alpha'),
      ('project.protectedAreas.failed/v1/alpha');
```